.circleci/config.yml

```diff
@@ -12,28 +12,37 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
+            - v2-dependencies-{{ checksum "package.json" }}
```

Comment:
This is because of the swap to npm; bumping the key version to keep it from pulling in any old stuff it cached under yarn